sound.js

```diff
@@ -159,9 +166,17 @@ Sound.prototype.setNumberOfLoops = function(value) {
 
 Sound.prototype.setSpeed = function(value) {
   this._setSpeed = value;
+  this._speed = value;
   if (this._loaded) {
-    if (!IsWindows) {
-      RNSound.setSpeed(this._key, value);
+    if (!IsWindows && !IsAndroid) {
+        RNSound.setSpeed(this._key, value);
+    }else if (IsAndroid) {
+      const isPlayingPromise = RNSound.isPlaying(this._key);
+      isPlayingPromise.then((result) => {
```

Comment:
This has several problems:

- The previous implementation was synchronous, however, how this function became an asynchronous function. Even worse, this library works asynchronously depending on the condition. 

-  Any public asynchronous function should return `Promise` object because it provides an interface to chain further tasks after this. WIthout it, the other developer loses the control of the subsequent code. Please refer `Sound.prototype.isPlaying()` regarding how to deal with this cases.

- Lastly, changing this method to asynchronous version could break the backward compatibility. Please note that this library is basically designed to be the public use, so we also need to care about other developers. If we do not consider this situation, we might not be able to merge our changes to upstream.

Thus, there should be an alternative way to handle this issue. I'll continue to comment by tonight.

Reply:
@ghsdh3409 I also ask for your feedback here.

Reply:
I fixed it to not use built in isPlaying function, instead, manage its own `this._playing` variable to check if the playing is being played. Would this cause problem too?